depends/packages/expat.mk

```diff
@@ -9,6 +9,7 @@ define $(package)_set_vars
   $(package)_config_opts += --disable-dependency-tracking --enable-option-checking
   $(package)_config_opts += --without-xmlwf
   $(package)_config_opts_linux=--with-pic
+  $(package)_cflags += -fno-lto
```

Comment:
I'm pretty sure the https://github.com/libexpat/libexpat/commit/cf3c2aa0842187495ebe5f1bdb5f4f66c6ec3eff commit causes this problem.